.azuredevops/pipelineTemplates/jobs.validateModuleDeployment.yml

```diff
@@ -180,7 +180,18 @@ jobs:
 
             Write-Verbose "Convert Tokens Input:`n $($ConvertTokensInputs | ConvertTo-Json -Depth 10)" -Verbose
 
-            # Invoke Token Replacement Functionality
```

Comment:
Good call. Just one question: Should we then remove the token replacement from the [Get-DependencyResourceNameList.ps1](https://github.com/Azure/ResourceModules/blob/main/utilities/pipelines/resourceRemoval/helper/Get-DependencyResourceNameList.ps1) script?

Reply:
I would say not now, as we need to support using the script outside of the pipeline (local testing). If we embed removal feature into the current Test Module Locally script, then yes. Thoughts?

Reply:
I don't believe the script is used in the local test as it is only relevant for the resource removal - which is not part of the local testing

Reply:
Correct. The reason I purposely didn't remove it is to maintain existing functionality of the script as it can be used outside of the pipeline. Since it performs token replacement.